blindsign/blindrsa/blindrsa.go

```diff
@@ -144,6 +147,7 @@ func verifyBlindSignature(pub *rsa.PublicKey, hashed, sig []byte) error {
 func (state RSAVerifierState) Finalize(data []byte) ([]byte, error) {
 	kLen := (state.verifier.pk.N.BitLen() + 7) / 8
 	if len(data) != kLen {
+		fmt.Println(len(data))
```

Comment:
Presumably this is a debug line that you meant to delete?

Reply:
Uh, yes, oops.